test_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+set -e
+
+# Setup env
+INSTALL_PREFIX=`cat /tmp/install_prefix`
+mkdir -p nested/run
+cd nested/run
+python3.7 -m venv env
+source env/bin/activate
+python -m pip install -U -r $INSTALL_PREFIX/bin/requirements.txt
+pip freeze > $INSTALL_PREFIX/bin/requirements.txt
+
+# Start ephemeral network in the background
+timeout --signal=SIGINT --kill-after=30s --preserve-status 30s \
+python $INSTALL_PREFIX/bin/start_network.py \
+    -p ../../build/liblogging \
+    -b $INSTALL_PREFIX/bin \
+    -g $(pwd)/../../src/runtime_config/gov.lua \
+    -v &
+
+# Issue tutorial transactions to ephemeral network
+sleep 20
+python ../../python/tutorial.py client_info.txt
```

Comment:
Minor: This is only run in the daily CI. I guess we run it for every release, so the released docs are definitely good. Is it worth adding a `ctest` instance that runs this in the normal CI though, for easier repro/validation?

Reply:
I'm tempted to say let's not run it in `ctest` as I don't really want to have to think about/see this script during my daily CCF dev experience? Also, what it tests is definitely already tested by `e2e_logging`.

Reply:
Fair!